# HG changeset patch
# Parent  5b607d851a9c8d71384dd29984f629f18f3366ca
Don't abort on request without an id in debug builds

We made it optional.

diff --git a/core/jsonrpc.c b/core/jsonrpc.c
--- a/core/jsonrpc.c
+++ b/core/jsonrpc.c
@@ -1118,7 +1118,6 @@
     }
 
     assert(request.method);
-    assert(request.id);
 
     for (int i = 0; i != LGTD_ARRAY_SIZE(methods); i++) {
         int parsed_method_namelen = LGTD_JSONRPC_TOKEN_LEN(request.method);
